src/public/less/pages/documentation.less

```diff
@@ -0,0 +1,337 @@
+@scrn-md: 768px;
+
+.p-documentation {
```

Comment:
Not sure about this file - is it all unique styles or how much are we duplicating the policy styles? If needed, we can use a more generic class name for the shared styles.

Reply:
Terms page is broken, need to be fixed. Because the Homepage and package page is in priority i will fix Terms right after them with the usage of Docs page styles.